app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  include PgSearch::Model
+
+  pg_search_scope :search_by_title, against: :title
```

Comment:
This was totally unnecessary, but a fun little "spice" I tossed on top (literally) 🙈